### PR TITLE
Mojtaba openfoam simulation

### PR DESCRIPTION
**Pull request by mojtabaAmiraslanpour**
Hi, I have completed the first draft to the simulation section of OpenFOAM. @teymourj you may review the changes and make needed probable corrections.
Many thanks.

### CODE REVIEW

**Comment by teymourj** [comment]
@mojtabaAmiraslanpour excellent job on using the previously developed template and adapting it to your case study. There were only few minor comments on your content that were implemented directly (this was done to save your time upon the previous conversations).

However, there was a comment that needs your confirmation, please see the comment below within the body of the text and confirm or comment on the change. Upon your approval the file will be push on the master branch.

File: Open_Foam/Simulation.ipynb [hunk 1]
```diff
@@ -5,199 +5,233 @@
...
+    "\t    timeInterval    10;\n",
+    "\n",
+    "\t    log         no;\n",
+    "\n",
+    "\t    patches     ( cylinder );\n",
+    "\t    pName       p;\n",
+    "\t    UName       U;\n",
+    "\t    rhoName     rhoInf;  // Indicates incompressible\n",
+    "\t    log         true;\n",
+    "\t    rhoInf      100;     // Redundant for incompressible\n",
```
Comment (teymourj) [inline Open_Foam/Simulation.ipynb]: The only comment that needs your approval is here on the settings for the "forcesCoeffs": the set reference density was 1000, however the set kinematic viscosity was 0.01 (i.e. representing Re=100 and density of 100 [kg/m^3]). The value of reference was changed to 100. Please confirm the validity of this change.

Comment (mojtabaAmiraslanpour) [inline Open_Foam/Simulation.ipynb]: Actually for incompressible solvers the value of density is redundant and wont effect the results. In these solvers the momentum equation is divided by rho and therefore setting the rho to any value would make no changes. You may also mention that the dimension of pressure in 0/p file is set to m^2/s^2 which is the dimension for pressure divided by density. As a result there is no difference between 1000 or 100 in here. The only thing that matters is the nu value which is set to be 0.01 resulting in Re number of 100.

Comment (teymourj) [inline Open_Foam/Simulation.ipynb]: Great! This was elaborated at the corresponding section and density reference value was set to 1 for sake of consistency with other variables.
This pull-request will then be officially merged to the master branch now.

**Pull request merged**